src/js/letters/containers/LetterList.jsx

```diff
@@ -34,24 +26,19 @@ export class LetterList extends React.Component {
         content = letterContent[letter.letterType] || '';
       }
 
-      let downloadLetterLink;
-      if (this.props.optionsAvailable) {
-        downloadLetterLink = (
-          <DownloadLetterLink
-            letterType={letter.letterType}
-            letterName={letter.name}
-            downloadStatus={downloadStatus[letter.letterType]}
-            key={`download-link-${index}`}/>
-        );
-      }
-
       return (
         <CollapsiblePanel
           panelName={letterTitle}
           key={`collapsiblePanel-${index}`}>
           <div>{content}</div>
-          <div>{downloadLetterLink}</div>
-          <div>{bslHelpInstructions}</div>
+          <DownloadLetterLink
+            letterType={letter.letterType}
+            letterName={letter.name}
+            downloadStatus={downloadStatus[letter.letterType]}
+            key={`download-link-${index}`}/>
+          { letter.letterType === LETTER_TYPES.benefitSummary &&
```

Comment:
I guess this is a style difference, but I liked how before all the checks for the types of letters and the condition logic that rendered because of that was in one place up above. Now we're checking in 2 places if `letter.letterType === LETTER_TYPES.benefitSummary`. And is that slightly more efficient because we're only doing that check once? Thoughts?